homeassistant/components/modbus/modbus.py

```diff
@@ -419,9 +419,15 @@ def pb_call(
         except ModbusException as exception_error:
             self._log_error(str(exception_error))
             return None
+        if not result:
+            self._log_error("Error: pymodbus returned None")
+            return None
         if not hasattr(result, entry.attr):
             self._log_error(str(result))
             return None
+        if not result.isError():  # type: ignore[no-untyped-call]
```

Comment:
```suggestion
        if result.isError():  # type: ignore[no-untyped-call]
```

This looks reversed?

Reply:
I ned coffee !!!!